### PR TITLE
fix(annotations): add annotations information as a url input to this npm

### LINKED ISSUE #106: Implement Loader during the stack analysis API call
It sometime takes a while for the network to respond and it is always good to have a loader in such times.

### PR DESCRIPTION
**Pull request by arunkumars08**
Changes:
1. Changes the app.component.* files to make it run in the standalone
2. Changes in the stack-analysis-service, stack-details.component.html, ts
3. Changes the modal build title
4. Introduces loader image and the feature
5. Changes in the stack-details.component.scss file to style loader
6. Hide and show logic for loader in stack-details.component.ts file
7. Changes in the gulpfile to compile and move images to dist during build
8. Changes in the style of 'Stack Reports' from button to link.

Fixes #106.

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1 +1 @@
-<stack-details [stack]="codebases && codebases[0]"></stack-details>
\ No newline at end of file
+<stack-details [stack]="stackUrl && stackUrl"></stack-details>
```
Comment (invincibleJai) [inline src/app/app.component.html]: can we have just `[stack]="stackUrl"`

Comment (invincibleJai) [inline src/app/app.component.html]: missing new line at file end

Comment (arunkumars08) [inline src/app/app.component.html]: Yes. It is just an extra check. I will handle the EOL. I think that check can be there.

**Comment by arunkumars08** [comment]
Thanks, @invincibleJai  Merging into master

**Pull request merged**